apps/daimo-mobile/src/sync/sync.ts

```diff
@@ -237,10 +238,10 @@ function applySync(account: Account, result: AccountHistoryResult): Account {
 }
 
 export function syncFindSameOp(
-  op: TransferOpEvent,
+  opHash: Hex,
```

Comment:
nit but i'd do
`opHash?: Hex` here

then
```
if (opHash == null) return null
```

then you dont need the `!` above

& HistoryOpScreen can be just

```ts
op = syncFindSameOp(op.opHash, account.recentTransfers) || op;
```

Reply:
done